testing/trino-benchto-benchmarks/src/test/resources/hive_metadata/README.md

```diff
@@ -0,0 +1,7 @@
+These metadata files were generated from the glue connector from TPCDS/TPCH schema with sf1000.
+So, there are no min/max statistics for char-based columns.
```

Comment:
> So, there are no min/max statistics for char-based columns.

Why?

Reply:
I don't think we support min/max statistics for `varchar` and `char` columns in the hive.

https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java#L961

Reply:
See `StatsUtil#toStatsRepresentation`, it's still the case, we rely more on NDV in that case